### ISSUE TITLE
[bug] Project and Project Tasks Priority greyed out

### ISSUE THREAD

**Issue by deanathpc** [YetiForceCompany/YetiForceCRM#13258]
I seem to be having an issue like a user has had previously.

While editing Projects and Project Tasks I am unable to change the Priority of each.  I finally was able to turn on my permissions for the Project Milestone priory so that now works.  But the other two do not.

I have gone in to Software configuration | Standard modules | Fields – Picklists | System settings

Project Tasks | Priority and selected | Values Assigned to a Role | Role Name and selected Board Of Management then selected all three and saved.

Projects | Priority and selected | Values Assigned to a Role | Role Name and selected Board Of Management then selected all three and saved.

Priority Box is greyed out.  if I refreshed the page for a moment I can see a down arrow representing a combobox to select an item but it quickly goes away.  Stays greyed out.

Using version 5.3.0.  This is under an Administrator account also.

**Comment by blu-IT** [comment]
Can you please post a screenshot of your settings:

```
Project Tasks | Priority and selected | Values Assigned to a Role | Role Name and selected Board Of Management then selected all three and saved.

Projects | Priority and selected | Values Assigned to a Role | Role Name and selected Board Of Management then selected all three and saved.
```

**Comment by deanathpc** [comment]
Here are the screenshots you asked for.


![YetiForce-Project Tasks](https://user-images.githubusercontent.com/67805826/86521061-cd892b00-be19-11ea-8460-479141007f9a.PNG)
![Yetiforce-Projects](https://user-images.githubusercontent.com/67805826/86521065-cfeb8500-be19-11ea-92c2-3a1d78a8d0ca.PNG)

**Comment by blu-IT** [comment]
Have you checked, that the Administrator account you are using is inside group "Board of Management"?

**Comment by deanathpc** [comment]
> 
> 
> Have you checked, that the Administrator account you are using is inside group "Board of Management"?

Yes.  Both accounts are "Board of Management" and I even switched one to another account and gave permissions and same result.

**Comment by blu-IT** [comment]
> I seem to be having an issue like a user has had previously.

I had that issue (with tickets) and it came with the transfer of database to a new installation. The rights did not correspond any more. But you already checked that above and form the screenshots everything seems OK.
Is it a new installation? Did you migrate the database to a new host? Has there been a crash of the machine and the database may be damaged?

Have you tried to create a new Project, Project Task or even Ticket to see if this behavior still persists?

**Comment by vovpff** [comment]
#13105 can be solution

**Comment by deanathpc** [comment]
> 
> 
> > I seem to be having an issue like a user has had previously.
> 
> I had that issue (with tickets) and it came with the transfer of database to a new installation. The rights did not correspond any more. But you already checked that above and form the screenshots everything seems OK.
> Is it a new installation? Did you migrate the database to a new host? Has there been a crash of the machine and the database may be damaged?
> 
> Have you tried to create a new Project, Project Task or even Ticket to see if this behavior still persists?

This is a new installation.  So no migration or anything like that.  No crash of the machine that I am aware of.  This has been happening since first installed.

I had other fields that were doing the same in different areas and once I changed permissions all is well so far.  Project Milestone was doing the same but once I changed those permissions everything is working there too.

**Comment by deanathpc** [comment]
> 
> 
> #13105 can be solution

What part of that?  the SQL query part you mean?  That link is about paymentsin though.  I'm working with projects.

**Comment by vovpff** [comment]
Project priority can be stored in database as high instead PLL_HIGH, in this case you can't change this field value. The same situation was with payments in status value

**Comment by deanathpc** [comment]
> 
> 
> Project priority can be stored in database as high instead PLL_HIGH, in this case you can't change this field value. The same situation was with payments in status value

Ahh O.K.  Well they all show as low in each record if that makes a difference?

So how does one fix it?  With the SQL query statement?

**Comment by deanathpc** [comment]
> 
> 
> Project priority can be stored in database as high instead PLL_HIGH, in this case you can't change this field value. The same situation was with payments in status value

Just checked a few records.  Stored as Low.  Looks like all of them are.

I manually chanced it in the database and it comes up with what I plugged in.  So it's changeable.  But I am unable to change it via Yetiforce.

**Comment by deanathpc** [comment]
> 
> 
> Project priority can be stored in database as high instead PLL_HIGH, in this case you can't change this field value. The same situation was with payments in status value

Actually you bring up an excellent point.  So in my database it's in there as just Low.  I see in other tables it's like you described PLL_LOW or PLL_HIGH.

So YetiForce isn't putting the data in correctly.  Off to check some other tables.

**Comment by deanathpc** [comment]
I also just checked the two tables for Task Priority and Project Priority.  They both had entries of low, normal, high.  Not PLL_LOW, PLL_NORMAL or PLL_HIGH.

**Comment by vovpff** [comment]
Rename language folder to _language and go to pick list edit page. Check there correct picklists values?

**Comment by deanathpc** [comment]
> 
> 
> Rename language folder to _language and go to pick list edit page. Check there correct picklists values?

It shows just low, normal, high.  Not PLL_LOW, PLL_NORMAL or PLL_HIGH.  If that is what you are asking.

Just for grins and giggles I checked the help desk section.  Support. Created a new ticket.  That priority is working correctly.  And is labeled as Low, Normal, High and Urgent.  And that combo box works just fine.

Is the fact it is all lower case in my problem areas?  Or does that not matter?

**Comment by vovpff** [comment]
@deanathpc I has also checked picklist values in my system. All looks perfect.

Check this 

![image](https://user-images.githubusercontent.com/8545791/86790551-3cac8c80-c071-11ea-813e-7c102f852b74.png)

**Comment by blu-IT** [comment]
Can you try to uncheck "Default value" low and see if it changes something?

**Comment by vovpff** [comment]
@deanathpc 

![image](https://user-images.githubusercontent.com/8545791/86796586-6b2d6600-c077-11ea-8d49-b0a4ee496284.png)

Default value `Low` instead `low`. Uncheck default value. Save. After that check default value and select low.

@rskrzypczak pls take a look!

**Comment by deanathpc** [comment]
> 
> 
> @deanathpc I has also checked picklist values in my system. All looks perfect.
> 
> Check this
> 
> ![image](https://user-images.githubusercontent.com/8545791/86790551-3cac8c80-c071-11ea-813e-7c102f852b74.png)

I can't seem to find this.  How does one get to that edit field?

**Comment by deanathpc** [comment]
Nevermind.  I found it.  Sorry..

**Comment by deanathpc** [comment]
Here is what my edit field box looks like.

![YF-EditField](https://user-images.githubusercontent.com/67805826/86822380-9012e080-c059-11ea-91a0-3e709c76eb38.PNG)

So I unchecked Default.  No change.  Re-checked Default and selected "low" and no change.

I also tried a fresh in stall on my webhost.  In another directory fresh database.  The same problem is on the fresh install also.  Softaculous.  I use bluehost.com for my webhost.

**Comment by deanathpc** [comment]
WAIT!!!!!!!!!!!!!!!!!!!!!!!!

I just added a new record!!!!  I can select priority now!!!!  But edit a record and I can't.  ????????

**Comment by deanathpc** [comment]
O.K. I'm on to something.

I am sorry for disrupting your troubleshooting flow but I stepped out of the box here.

I just went and renamed the Project Tasks Priority entries.  low to Low, normal to Normal and high to High.  I then changed the Default value in the Edit Field box.

Bingo.  I can start a new record and I have my priority options.  I can now edit a previous record and BINGO I can edit those now too.

So this comes down to a case of the system being picky about case values.  Capital letters is key here.

Does this help?
